pkg/cri/opts/spec_test.go

```diff
@@ -44,3 +47,29 @@ func TestOrderedMounts(t *testing.T) {
 	sort.Stable(orderedMounts(mounts))
 	assert.Equal(t, expected, mounts)
 }
+
+func TestDriveMounts(t *testing.T) {
+	tests := []struct {
+		mnt                   *runtime.Mount
+		expectedContainerPath string
+		expectedError         error
+	}{
+		{&runtime.Mount{HostPath: "C:\\", ContainerPath: "D:\\foo"}, "D:\\foo", nil},
```

Comment:
I'm good with this but on other PR's I've seen feedback to use literals so `C:\` `D:\foo` rather than `\\`

Reply:
Do me a solid and add a named pipe mount too to valid that fork of the code.

Reply:
+1 to using raw strings (backticks) so that we don't need to escape backslashes.